controllers/articles.contoller.js

```diff
@@ -0,0 +1,16 @@
+const {fetchArticleById} = require('../models/article.model.js')
+
+exports.getArticleById = (req, res, next) => {
+    const id = req.params.article_id
+
+    if (isNaN(id)) return next({ 
+        status: 400, 
+        message: 'bad request: this is not a number'
+    })
```

Comment:
This isn't needed as if the article ID is not a number then PSQL would throw an error which we could then handle with an error handler.

Seperatley if we did want to keep this in, we would want to send a Promise.reject ideally from the model instead of invoking next.